src/test/java/com/speaker/service/AccountServiceImplTest.java

```diff
@@ -88,4 +95,49 @@ public void accountNotCreatedWhenCityNameIsNull() {
         AccountDTO accountDTO = generateAccountDTO(generateCountryDTO(CountryName.UKRAINE, generateCityDTO(null)));
         assertThat(accountService.create(accountDTO), is(Response.FALSE));
     }
+
+    @Test
+    public void addFriendSuccess() {
+        FriendDTO friendDTO = generateFriendDTO("Ivan Mazur", "Victor Mazur");
+        Friend friend = generateFriend(1, 2);
+        when(accountRepository.findAccountIdByNameAndLastName("Ivan", "Mazur")).thenReturn(Optional.of(1));
+        when(accountRepository.findAccountIdByNameAndLastName("Victor", "Mazur")).thenReturn(Optional.of(2));
+        when(friendConverter.convertToFriend(1, 2)).thenReturn(friend);
+        when(accountRepository.addFriend(friend)).thenReturn(1);
+        assertThat(accountService.addFriend(friendDTO), is(Response.TRUE));
+    }
+
+    @Test
+    public void addFriendSuccessWithSpaces() {
+        FriendDTO friendDTO = generateFriendDTO("  Ivan   Mazur  ", " Victor  Mazur  ");
+        Friend friend = generateFriend(1, 2);
+        when(accountRepository.findAccountIdByNameAndLastName("Ivan", "Mazur")).thenReturn(Optional.of(1));
+        when(accountRepository.findAccountIdByNameAndLastName("Victor", "Mazur")).thenReturn(Optional.of(2));
+        when(friendConverter.convertToFriend(1, 2)).thenReturn(friend);
+        when(accountRepository.addFriend(friend)).thenReturn(1);
+        assertThat(accountService.addFriend(friendDTO), is(Response.TRUE));
+    }
+
+    @Test
+    public void addFriendWhenAccountNameNull() {
+        FriendDTO friendDTO = generateFriendDTO(null, "Victor Mazur");
+        assertThat(accountService.addFriend(friendDTO), is(Response.FALSE));
+    }
+
+    @Test
+    public void FriendNotCreatedWhenAccountNameHasThreeWord() {
```

Comment:
the method name must start in lower case

Reply:
Updated. Changes made